.travis.yml

```diff
@@ -1,15 +1,16 @@
 language: julia
 os:
     - linux
+    - osx
 julia:
     - 0.5
     - nightly
 notifications:
     email: false
-script:
-    - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-    - julia -e 'Pkg.clone(pwd()); Pkg.clone("Winston")'
-    - xvfb-run julia -e 'Pkg.build("Tk"); Pkg.test("Tk"; coverage=true)';
-    # - xvfb-run julia -e 'Pkg.clone(pwd()); Pkg.build("Tk"); Pkg.test("Tk"; coverage=true)';
+#script:
+#    - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+#    - julia -e 'Pkg.clone(pwd()); Pkg.clone("Winston")'
+#    - xvfb-run julia -e 'Pkg.build("Tk"); Pkg.test("Tk"; coverage=true)';
```

Comment:
might need to put this back conditionally later, a few other packages do this where you check `uname` or `TRAVIS_OS_NAME` to use `xvfb-run julia` on linux or just `julia` on mac